File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/VolunteerRequestResponseDto.java
package com.sahayaSetu.dtos;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class VolunteerRequestResponseDto extends RequestResponseDto {
    private String skillsRequired;
    private int volunteersRequired;
    private long volunteerCount;
    private boolean hasApplied = false;
}

File: backend-dotnet/SahayaSetu/DTO/VolunteerRequestResponseDto.cs
using SahayaSetu.Models;

namespace SahayaSetu.DTO
{
    public class VolunteerRequestResponseDto : RequestResponseDto
    {
        public string SkillsRequired { get; set; } = string.Empty;
        public int VolunteersRequired { get; set; }
        public long VolunteerCount { get; set; }
        public bool HasApplied { get; set; }
    }
}
